### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in Claims Status end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7313]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/claims-status` app folder, I found one test that needs additional coverage. I'd like the front-end engineering team to review this application, and add end-to-end tests as needed. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/claims-status/tests`
* `$ yarn test:e2e src/applications/claims-status/tests`

## Acceptance Criteria

**Definition of done:**
- [x] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [x] axe check is added to `05.claim-additional-evidence.e2e.spec.js`, preferably just before closing the page
- [x] axe checks are run for hidden content like modal windows, accordions
- [x] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [x] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)
